doc/source/serve/resource-allocation.md

```diff
@@ -0,0 +1,68 @@
+(serve-cpus-gpus)=
+
+# CPU & GPU support and Customized resource
```

Comment:
```suggestion
# CPU and GPU support and custom resources
```

Reply:
"and" is preferred over "&" here (I had to google it).  Also changing to "custom resources" for consistency with the rest of the doc